modules/core/src/test/java/org/apache/ignite/internal/ClusterBaselineNodesMetricsSelfTest.java

```diff
@@ -82,9 +92,13 @@ public void testBaselineNodes() throws Exception {
         log.info(String.format(">>> State #1: topology version = %d", ignite0.cluster().topologyVersion()));
 
         assertEquals(2, mxBeanCluster.getTotalServerNodes());
+        assertEquals(2, ((IntMetric)mreg.findMetric("TotalServerNodes")).value());
```

Comment:
Let's use `merg.<IntMetrics>findMetric("XXX").value()` to avoid type casting.

Reply:
Fixed.